### PR TITLE
Page titles component

### PR DESCRIPTION
**Pull request by eledwinn**
## Summary
- Added sizes.scss file for size variables
- Added page title component
- Added assets folder and files

## Screenshot
image

## Trello Card
https://trello.com/c/Pq2xqc9N/21-page-titles

### CODE REVIEW

File: scss/_sizes.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
```
Comment (CarlosHWolox) [inline scss/_sizes.scss]: and empty line at end of file

File: scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+
+
+@import '_sizes';
```
Comment (CarlosHWolox) [inline scss/application.scss]: Delete all this empty space
Also add empty line at end of file
You should import your equalizer at the top of yout application css file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  <meta name="author" content="Wolox">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="tech-title">
+    <img class="title-decoration" src="assets/triangle.svg" alt="triangle" >
+    <p>Front-End Tools</p>
+  </div>
```
Comment (CarlosHWolox) [inline index.html]: There is a simpler way to do this, we try to use html only for really important structural elements, decoration elements like this image can be added throught CSS, use a ::before css pseudoclass to add this. With that change you could delete the div and the p tag and emphasize your title with html also adding it a header title that goes better with the title, like and h1, h2, h3 ......

File: scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  .title-decoration {
+    height: 90px;
+    width: 70px;
+  }
+
+  p {
+    font-size: $text-big !important;
+    text-indent: -18px;
+  }
+}
```
Comment (CarlosHWolox) [inline scss/titles.scss:16]: Read the changes suggested on the index file and modify this ones.

File: scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.tech-title {
+  align-items: center;
+  display: flex;
+  font-size: $text-big !important;
```
Comment (CarlosHWolox) [inline scss/titles.scss]: Avoid the use of `!important`

File: scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
+.tech-title {
+  align-items: center;
+  display: flex;
```
Comment (CarlosHWolox) [inline scss/titles.scss]: why the flex?

File: scss/titles.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  position: relative;
+  
+  &::before {
+    background: url(assets/triangle.svg);
+    background-size: 70px 90px;
+    content: '';
+    left: 0;
+    height: 100%;;
+    position: absolute;
+    width: 70px;
```
Comment (CarlosHWolox) [inline scss/titles.scss:13]: width and height are really neccesary?

Comment (eledwinn) [reply scss/titles.scss:13]: Is not neccesary, replaced by top,left,bottom,right properties.

File: scss/titles.scss (hunk 6 above)
Comment (eledwinn) [reply scss/titles.scss]: I used flex for center text vertically, I could have used line-height, but I preferred to use flex. 
Any suggestions?

File: scss/titles.scss [hunk 8]
```diff
@@ -0,0 +1,20 @@
+.tech-title {
+  align-items: center;
+  display: flex;
+  font-size: $text-big;
+  height: 90px;
+  padding-left: 52px;
```
Comment (CarlosHWolox) [inline scss/titles.scss]: Instead of padding the main element is better to move the decorative one.

Comment (eledwinn) [reply scss/titles.scss]: Padding in the main element is removed and translateX is added in background image.

**Review by CarlosHWolox — APPROVED** [review]

**Pull request merged**